### ISSUE TITLE
Padding layer error Error Code 2: Internal Error (Assertion rank == 0 || rank == 1 failed. )

### ISSUE THREAD

**Issue by jackgao0323** [NVIDIA/TensorRT#2083]
## Description

I'm using the trtexec to parsing the onnx model and got the error as title. What's the 'rank' mean and how could I fixed this error?

The trtexec command I used is:

> trtexec.exe --onnx=C:\Users\jackg\Downloads\project\model\Depth\test_modified.onnx --saveEngine=C:\Users\jackg\Downloads\project\model\Depth\engine\test_fp16.trt --minShapes='input':1x6x480x640 --optShapes='input':1x6x480x640 --maxShapes='input':1x6x480x640 --verbose --shapes='input':1x6x480x640 --fp16

Here is the error image.
<img width="946" alt="image" src="https://user-images.githubusercontent.com/17614632/175234184-3cf759c9-d271-4a14-bc30-d8cacfa5a1ae.png">

## Environment

**TensorRT Version**: 8.4.1.5
**NVIDIA GPU**: 3080
**NVIDIA Driver Version**: 516.40
**CUDA Version**: 11.1
**CUDNN Version**: 8
**Operating System**: windows 10
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files

Here is the onnx model. I convert it from tensorflow by using tf2onnx.
[test_modified.zip](https://github.com/NVIDIA/TensorRT/files/8964731/test_modified.zip)

**Comment by nvpohanh** [comment]
This is because the "pads" of the padding op uses complicated computations to compute how much to pad, but that part requires N-D shape tensor which TRT will not support until next TRT version. For now, you could try const-folding the ONNX file using https://github.com/NVIDIA/TensorRT/tree/main/tools/Polygraphy/examples/cli/surgeon/02_folding_constants or set the H and W to static shape when exporting the ONNX model. Thanks

![2022-06-24 12_25_30-C__Users_phuan_AppData_Local_Temp_Temp1_test_modified zip_test_modified onnx - N](https://user-images.githubusercontent.com/53919306/175462061-053c0715-7529-428c-aed4-b9679c9c2872.png)

**Comment by jackgao0323** [comment]
I've tried folding constant  but still don't work. I think it's not support now. I think I'll use the static shape.

**Comment by vedanshthakkar** [comment]
I have tried constant folding (polygraphy) and also used the static shape. The error is still there. 

Tensorflow-gpu=2.4.0
keras=2.4.0
tensorrt = 8.4.3.1
os = windows10

Is there a solution?
